tests/libgstc/test_libgstc_client.c

```diff
@@ -70,9 +71,27 @@ gstc_json_is_null (const gchar * json, const gchar * name, gint * out)
 }
 
 GstcStatus
-gstc_json_get_child_char_array(const char *json, const char* parent_name,
-  const char* array_name, const char *element_name, char **out[], int *array_lenght)
+gstc_json_get_child_char_array (const char *json, const char *parent_name,
+    const char *array_name, const char *element_name, char **out[],
+    int *array_lenght)
+{
+  gstc_assert_and_ret_val (NULL != json, GSTC_NULL_ARGUMENT);
```

Comment:
I usually don't care about this in tests.

Reply:
As discussed here: https://github.com/RidgeRun/gstd-1.x/pull/28#discussion_r208628909 it might be better for maintainability